sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2684,11 +2684,17 @@ object SQLConf {
       .doc("When true, NULL-aware anti join execution will be planed into " +
         "BroadcastHashJoinExec with flag isNullAwareAntiJoin enabled, " +
         "optimized from O(M*N) calculation into O(M) calculation " +
-        "using Hash lookup instead of Looping lookup." +
-        "Only support for singleColumn NAAJ for now.")
+        "using Hash lookup instead of Looping lookup.")
```

Comment:
Add few words to link this with OPTIMIZE_NULL_AWARE_ANTI_JOIN_MAX_NUM_KEYS. E.g., "The number of keys supported for NAAJ is configured by spark.sql.optimizeNullAwareAntiJoin.maxNumKeys".

Reply:
done